crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -765,6 +767,56 @@ mod test {
         assert_eq!(ev_id(client_room.latest_event()), "$idb");
     }
 
+    #[async_test]
+    async fn cached_latest_event_can_be_redacted() {
```

Comment:
Most tests for `latest_event` seem to live in `matrix-sdk-base`, would it make sense to move this one there as well?

Reply:
This *is* `matrix-sdk-base`.

Reply:
Please never mind my hallucinations.